projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
and I am sure Alex is the one who wrote this, back in time, everyone c/p'd license headers when contributing to OE. so Stephen Raue is listed everywhere, but Alex is the author. if you are unsure, dont touch it :)